feature/emergency-stop

MarkRoss-Eviden opened the pull request:
Added emergency stop functionality to reboot the DeepRacer service.

Ideal in the event of the car getting 'stuck' doing the same action (e.g. going round in circles), use the emergency stop button restarts the service and once the service restarts the car should be ready to use again, whereas before a full power cycle of the vehicle was required.

Pre-requisite for this functionality to work is the AWS DeepRacer custom car code - https://github.com/aws-deepracer-community/deepracer-custom-car.  If the required API isn't available then the button is hidden from the UI.

Update UI looks like this -
![image](https://github.com/user-attachments/assets/5d85b153-40d1-4ff9-b670-9623469635d1)

MarkRoss-Eviden commented:
@larsll thought you may be opinionated on look of UI, any feedback?

larsll commented:
> @larsll thought you may be opinionated on look of UI, any feedback?

Yeah -- the font is massively off! :)

MarkRoss-Eviden commented:
Updated following feedback: -

Change button font to Open Sans
Added note of caution to -5 / +5 buttons
Removed STOP! Button from model control page
Added Emergency Stop Button to the navigation panel

Buttons before model loaded: -
![image](https://github.com/user-attachments/assets/b3e56b34-849a-4529-bf5d-26ccb1c61cde)


Buttons after model loaded: -
![image](https://github.com/user-attachments/assets/037e12f1-c332-473c-a77e-a2867d14e2f2)


Nav panel with custom car code emergency stop: -
![image](https://github.com/user-attachments/assets/ac22ed7c-ec42-40ee-82e1-40ece1e5dcbc)

Nav panel without custom car code emergency stop: -
![image](https://github.com/user-attachments/assets/34c229c5-8d43-4375-8d8d-34f4e9c2671d)
